flycheck.el

```diff
@@ -3945,30 +3951,40 @@ show the icon."
     (error "Invalid fringe side: %S" side))
   (propertize "!" 'display
               (list side
-                    (flycheck-error-level-fringe-bitmap level)
+                    (flycheck-error-level-fringe-bitmap level
+                                                        (= (car (window-fringes)) 16))
```

Comment:
This should be `car` or `cadr` depending on whether `side` is `left-fringe` or `right-fringe`:

```
(= 16  (pcase side (`left-fringe (car (window-fringes))) (`right-fringe (cadr (window-fringes)))))
```